src/tests/ftest/osa/osa_online_parallel_test.py

```diff
@@ -271,7 +278,13 @@ def run_online_parallel_test(self, num_pool):
             for val in range(0, num_pool):
                 display_string = "Pool{} space at the End".format(val)
                 pool[val].display_pool_daos_space(display_string)
-                pver_end = self.get_pool_version()
+                fail_count = 0
+                while fail_count <= 20:
+                    pver_end = self.get_pool_version()
+                    time.sleep(10)
+                    fail_count += 1
+                    if pver_end > 23:
```

Comment:
I'm not familiar with this work, but what are you expecting pver_end to be here? 24 or 25? If it's 24, the test will fail because of the self.assertTrue below. If it's 25, it'll pass. What's the difference? I read the code and the ticket description, but didn't get this, so you might want to describe this logic in a comment.

Reply:
The value should be 25 to pass. Since we are reintegrating two targets the final count will be 25. Any other values, it should fail.